### ISSUE TITLE
Windows 10 media integration not working

### ISSUE THREAD

**Issue by des2say** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#2384]
Windows10

Google-Play-Music-Desktop-Player-4.2.0

Clean install of windows 10 on a new atom x5 cherry Trail based windows tablet and installed Google-Play-Music-Desktop-Player-4.2.0 and the Windows 10 media widget the shows up when adjusting the volume does show up when using Google-Play-Music-Desktop-Player-4.2.0, and it is also not on the lock screen. I have verified it works with other media programs such as Spotify and Microsoft Groove. I have Google-Play-Music-Desktop-Player-4.2.0 installed on a windows 10 laptop with an i7 with no issues. Seems to be only the tablet. Not sure if it is windows connected stand by causing the issue or the atom chip set. I have disabled connected stand by and uninstalled and reinstalled Google-Play-Music-Desktop-Player-4.2.0  with no change.

**Comment by MarshallOfSound** [comment]
This service is optional and defaults to off in 4.2.0.  just enable it in settings

**Comment by des2say** [comment]
I have enabled it in settings. and restarted the whole system with no change.

**Comment by TheBinaryFox** [comment]
@des2say @MarshallOfSound 
I'm also having this issue. I recently reinstalled Windows on my desktop computer, and I'm having both this issue and #2404. The latter was solved by following the instructions, but I haven't found any solution to this one.

I have tried restarting Windows and reinstalling GPMDP and its settings, and neither have had any effect. If you (MarshallOfSound) are unable to reproduce this issue, Is there any way I can contribute to help solve this?

**OS:**
Windows 10 Pro (64-bit)
Build 14393.969
